lib/engine/game/g_1880/step/company_pending_par.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/company_pending_par'
+
+module Engine
+  module Game
+    module G1880
+      module Step
+        class CompanyPendingPar < Engine::Step::CompanyPendingPar
+          def process_par(action)
+            super
+            corporation = action.corporation
+            @log << "#{corporation.name} selects ABC building permit"
+            corporation.building_permits = 'ABC'
+          end
```

Comment:
Ugly hard coding, kept it for now as 99% of the time when launching BCR the player will pick ABC permit (vs BCD). 
I will fix it later to be a choose action instead